src/command_modules/azure-cli-role/azure/cli/command_modules/role/_help.py

```diff
@@ -181,18 +183,49 @@
 """
 helps['ad app permission grant'] = """
     type: command
-    short-summary: Grant an app permissions from another app
+    short-summary: Grant the app an API permission
     examples:
         - name: Grant a native application with permissions to access an existing API with TTL of 2 years
           text: az ad app permission grant --id e042ec79-34cd-498f-9d9f-1234234 --app-id a0322f79-57df-498f-9d9f-12678 --expires 2
 """
 helps['ad app permission list'] = """
     type: command
-    short-summary: List the app permissions
+    short-summary: List API permissions the application has requests
     examples:
         - name: List the OAuth2 permissions for an existing AAD app
           text: az ad app permission list --id e042ec79-34cd-498f-9d9f-1234234
 """
+helps['ad app permission add'] = """
+    type: command
+    short-summary: add an API permission
+    long-summary: invoking "az ad app permission grant" is needed to activate it
```

Comment:
Can we just expose a flag to grant the permission when it is added or grant it by default and expose  flag to add but NOT grant a permission?

Reply:
I debated for a while on this for whether or not CLI will do too much, as portal exposes a separate button of "grant permission" for people to commit that.
With your comment, let us just do it then since the grant is inevitable.
I will expose "--grant" for people to opt in. This keeps the option for people to bulk grant at one shot

Reply:
Based on our discussion I think it is fine the way it is. If people have a lot of asks we can revisit in the future.